src/Aspire.Hosting.Analyzers/AppHostAnalyzer.Diagnostics.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+
+namespace Aspire.Hosting.Analyzers;
+
+public partial class AppHostAnalyzer
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("MicrosoftCodeAnalysisReleaseTracking", "RS2008:Enable analyzer release tracking")]
```

Comment:
What does this mean and why can we just suppress it?

Reply:
Copied from dotnet/aspnetcore, no idea :smile:

Reply:
Should we enable this? @eerhardt @joperezr

Reply:
We've had these disabled in the ASP.NET Core repo since forever and I think we disabled them when we started shipping analyzers in the shared framework out of habit as well.

If we could enable them from the get-go here, I'd be in favor of it since it'll force better practices around documenting analyzers.

That being said, I can't comment on how good the infrastructure is with working with these analyzer releasing tracking documents. For example, I'm not sure if there are codefixes for these in the same way there are for PublicApi.txt files.

Reply:
Seems like it does and seems to work pretty well:

https://github.com/user-attachments/assets/e13fc10e-7281-4af5-a294-d63376224aee

Reply:
I went ahead and enabled this.